lib/cmd/fh3/admin/domains/check.js

```diff
@@ -23,8 +23,10 @@ module.exports = {
     return "/box/api/domains/check?domain=" + domain;
   },
   'method' : 'get',
-  'postCmd': function(params, cb) {
-    params._table = common.createObjectTable(params);
+  'postCmd': function(argv, params, cb) {
+    if (!argv.json) {
+      params._table = common.createObjectTable(params);
+    }
     return cb(null, params);
```

Comment:
@wtrocki Could you approve it and in the future, we make it as smarter? We already have a JIRA for it  FH-3650.

Also. It is making the people doing a workaround as https://github.com/fheng/mbaas-art/pull/78. Let's don't blocker than and after we can improve the way to do it.